### PR TITLE
Docs: last batch of cookbooks

### PR DESCRIPTION
**Pull request by cmills1095**

### CODE REVIEW

File: cookbooks/geomio/doc/geomio.md [hunk 1]
```diff
@@ -20,44 +27,61 @@ repository by executing the command
...
 
-or you can download geomIO [here]. You will then need to add the geomIO
+or you can download geomIO [here](https://bitbucket.org/geomio/geomio/downloads/).
+You will then need to add the geomIO
 source folders to your MATLAB path by running the file located in
 `/path/to/geomio/installation/InstallGeomIO.m`. An extensive documentation for
-how to use geomIO can be found [here][1]. Among other things, it explains [how
-to generate drawings in Inkscape] that can be read in by geomIO, which
+how to use geomIO can be found [here](https://bitbucket.org/). Among other things, it explains [how
+to generate drawings in Inkscape](https://bitbucket.org/#drawing) that can be read in by geomIO, which
```
Comment (gassmoeller) [inline cookbooks/geomio/doc/geomio.md:35]: This link cannot be right. Can you check the previous link?

File: cookbooks/finite_strain/doc/finite_strain.tex [hunk 2]
```diff
@@ -8,7 +8,7 @@ \subsubsection{Tracking finite strain}
 
 In many geophysical settings, material properties, and in particular the rheology, do not only depend
 on the current temperature, pressure and strain rate, but also on the history of the system.
-This can be incorporated in \aspect{} models by tracking history variables through compositional fields.
+This can be incorporated in ASPECT models by tracking history variables through compositional fields.
```
Comment (gassmoeller) [inline cookbooks/finite_strain/doc/finite_strain.tex:11]: Was this change on purpose? I dont think we need any changes to the .tex file?

File: cookbooks/finite_strain/doc/finite_strain.md [hunk 3]
```diff
@@ -68,13 +74,13 @@ selected as material model. As it is derived from the "simple"
...
 
-``` prmfile
+```{literalinclude} finite_strain.part.prm
 ```
 
 ```{figure-md} fig:finite_strain
 <img src="finite_strain.svg" style="width:75.0%" />
 
-Accumulated finite strain in an example convection model, as described in Section <a href="#sec:finite-strain" data-reference-type="ref" data-reference="sec:finite-strain">0.0.1</a> at a time of 67.6&#xA0;Ma. Top panel: Temperature distribution. Bottom panel: Natural strain distribution. Additional black crosses are the scaled eigenvectors of the stretching tensor <span class="math inline"><strong>L</strong></span>, showing the direction of stretching and compression.
+Accumulated finite strain in an example convection model, as described in {ref}`sec:cookbooks:finite_strain` at a time of 67.6&#xA0;Ma. Top panel: Temperature distribution. Bottom panel: Natural strain distribution. Additional black crosses are the scaled eigenvectors of the stretching tensor <span class="math inline"><strong>L</strong></span>, showing the direction of stretching and compression.
```
Comment (gassmoeller) [inline cookbooks/finite_strain/doc/finite_strain.md:83]: there is something off about the time number ("67.6&#xA0;Ma"). Some leftover html code?

File: cookbooks/sinker-with-averaging/doc/sinker-with-averaging.md [hunk 4]
```diff
@@ -73,138 +73,50 @@ time step for this benchmark. This is all modeled in the following input file
...
-
-```{figure-md}
-<img src="sinker-8-harmonic.png" style="width:14.0%" />
+```{figure-md} fig:sinker-with-averaging-pressure
+<img src="q2q1.*" style="width:84.0%" />
 
  Visualization of the pressure field for the &#x201C;sinker&#x201D; problem. Left to right: No averaging, arithmetic averaging, harmonic averaging, geometric averaging, pick largest, project to Q_1. Top: 7 global refinement steps. Bottom: 8 global refinement steps. The minimal and maximal pressure values are indicated below every picture. This range is symmetric because we enforce that the average of the pressure equals zero. The color scale is adjusted to show only values between p=-3 and p=3.
```
Comment (gassmoeller) [inline cookbooks/sinker-with-averaging/doc/sinker-with-averaging.md:112]: Leftover html formatting?

File: cookbooks/sinker-with-averaging/doc/sinker-with-averaging.md [hunk 5]
```diff
@@ -227,118 +139,35 @@ A final comment is in order. First, one may think that the results should be
...
+iteration numbers are also largely unaffected if any kind of averaging is used
+-- though they are far worse using the locally conservative discretization if
+no averaging has been selected. On the positive side, the visualization of the
+discontinuous pressure finite element solution makes it much easier to see that
+the true pressure is in fact discontinuous along the edge of the circle.
+
+```{figure-md} fig:sinker-with-averaging-pressure-q2q1iso
+<img src="q2q1plus.*" style="width:84.0%" />
+
+ Visualization of the pressure field for the &#x201C;sinker&#x201D; problem. Like {numref}`fig:sinker-with-averaging-pressure` but using the locally conservative, enriched Stokes element. Pressure values are shown in {numref}`tab:sinker-with-averaging-max-pressure-q2q1iso`.
```
Comment (gassmoeller) [inline cookbooks/sinker-with-averaging/doc/sinker-with-averaging.md:162]: html formatting

**Review by gassmoeller — COMMENTED** [review]
Nice, this looks largely good. I left a few small comments. I dont know if there is a better way to replace the html formatting (e.g. by markdown formatting), just take a look if there is a better option.

File: cookbooks/geomio/doc/geomio.md (hunk 1 above)
Comment (cmills1095) [reply cookbooks/geomio/doc/geomio.md:35]: Ah I see, I was just clicking the links in the pdf manual and it redirected me there. I'm guessing if I was logged into bitbucket the links would work as intended. Me being lazy!

File: cookbooks/finite_strain/doc/finite_strain.tex (hunk 2 above)
Comment (cmills1095) [reply cookbooks/finite_strain/doc/finite_strain.tex]: Definitely not

**Comment by cmills1095** [comment]
This is the last official batch of cookbooks, but since there are still 160-some errors when the html pages are built I'm planning to do another pull request cleaning up a few things and addressing errors.

**Review by gassmoeller — APPROVED** [review]
Looks ready to go when the testers are done. Thanks for the massive amount of work!

**Review by tjhei — APPROVED** [review]

**Comment by tjhei** [comment]
Thank you!
